recce/run.py

```diff
@@ -45,34 +45,47 @@ def load_preset_checks(checks: list):
         check_params = check.get('params', {})
         check_options = check.get('view_options', {})
 
-        if check_type == RunType.SCHEMA_DIFF.value:
-            is_check = schema_diff_should_be_approved(check_params)
-
         create_check_without_run(name, description, check_type, check_params, check_options, is_preset=True,
                                  is_checked=is_check)
         table.add_row(name, check_type.replace('_', ' ').title(), description.strip())
     console.print(table)
 
 
 def schema_diff_should_be_approved(check_params: dict) -> bool:
-    context = default_context()
-    nodes = context.adapter.select_nodes(
-        select=check_params.get('select'),
-        exclude=check_params.get('exclude'),
-        packages=check_params.get('packages'),
-        view_mode=check_params.get('view_mode'),
-    )
-    nodes = [node for node in nodes if not node.startswith('test.')]
-    base = None
-    curr = None
-    for node in nodes:
-        base = context.get_model(node, base=True)
-        curr = context.get_model(node, base=False)
-    diff = DeepDiff(base, curr, ignore_order=True)
-
-    # If the diff is empty, then the check should be approved
-    if bool(diff) is False:
-        return True
+    try:
+        context = default_context()
+
+        if 'node_id' in check_params:
+            # If the node_id is provided, then use it
+            selected_node_ids = check_params.get('node_id', [])
```

Comment:
`node_id` can be `string` or `string[]`


`string` if it is added from NodeView
`string[]` if it is added from multi-node selection

Reply:
fix